tmt/steps/prepare/__init__.py

```diff
@@ -78,9 +79,9 @@ def go(
             logger.info('guest', guest.name, 'green')
 
         # Show requested role if defined
-        where = self.get('where')
+        where = cast(List[str], self.get('where'))
```

Comment:
The `FIXME` comment should be also here? :)

Reply:
Added in 08ae5788